### ISSUE TITLE
User not registered error

### ISSUE THREAD

**Issue by WickBRSTM** [samuelthomas2774/nxapi#123]
App normally works fine for me but when trying to show extra details while playing Splatoon 3, I keep getting this error:
![image](https://github.com/user-attachments/assets/7631b83e-7abf-4a8c-bd0e-c2eae2547892)
Any idea what's up? Both the main and alt account have Nintendo Switch Online.

**Comment by catgirlinspace** [comment]
does the alt account have access to SplatNet 3? (think you might need to play Splatoon 3 with it for it to be able to authenticate to SplatNet 3, but i'm not sure)

**Comment by WickBRSTM** [comment]
> does the alt account have access to SplatNet 3? (think you might need to play Splatoon 3 with it for it to be able to authenticate to SplatNet 3, but i'm not sure)

It should, but I'm actually not sure if I've opened the game on that account before. That might be it?

**Comment by jekkkk** [comment]
yeah i got the same thing, i just created a character and played a match and it works
